zip-0224.rst

```diff
@@ -7,14 +7,208 @@
           Sean Bowe <sean@electriccoin.co>
           Kris Nuttycombe <kris@electriccoin.co>
           Ying Tong Lai <yingtong@electriccoin.co>
-  Status: Reserved
+  Status: Draft
   Category: Consensus
   Discussions-To: <https://github.com/zcash/zips/issues/435>
 
-This ZIP number is reserved. Preliminary technical content relating to Orchard and Halo 2
-are currently in:
 
-* the `Orchard Book <https://zcash.github.io/orchard/design.html>`_;
-* the `Halo Book <https://zcash.github.io/halo2/>`_;
-* zcash/zcash GitHub tickets labeled `A-Orchard <https://github.com/zcash/zcash/issues?q=label%3AA-Orchard>`_
-  and `A-Halo <https://github.com/zcash/zcash/issues?q=label%3AA-Halo>`_.
+Abstract
+========
+
+This document proposes the Orchard shielded protocol, which defines a new shielded pool
+with spending keys and payment addresses that are amenable to future scalability
+improvements.
+
+
+Motivation
+==========
+
+TBD
+
+
+Specification
+=============
+
+The Orchard protocol is specified as an update to the Zcash Protocol Specification
+[#orchard-spec]_. Given that it largely follows the design of the Sapling protocol, we
+provide here a list of differences, with references to their normative specifications
+and associated design rationale.
+
+Curves
+------
+
+The Orchard protocol uses the Pallas / Vesta curve cycle, in place of BLS12-381 and its
+embedded curve Jubjub:
+
+- Pallas is used as the "application curve", on which the Orchard protocol itself is
+  implemented (c/f Jubjub).
+- Vesta is used as the "circuit curve"; its scalar field (being the base field of Pallas)
+  is the "word" type over which the circuit is implemented (c/f BLS12-381).
+
+We use (version 10 of) the IETF hash-to-curve Internet Draft [#ietf-hash-to-curve]_ to
+implement :math:`\mathsf{GroupHash}`, instead of the BLAKE2s-based mechanism used for
+Sapling. We specifically use the "simplified SWU" algorithm, which provides an infallible
+:math:`\mathsf{GroupHash}`.
+
+The presence of the curve cycle is an explicit design choice. This ZIP only uses half of
+the cycle (Pallas being an embedded curve of Vesta); the full cycle is expected to be
+leveraged by future ZIPs.
+
+- Curve specifications: [#spec-pasta]_
```

Comment:
Style question: should these be rendered as references, or in-line links? The latter would be easier to click through to, but I'm on the fence about which is clearer as to what references are normative vs informative.

Reply:
The Internet Draft is not normative (it's a draft, so it can't be). This part should say that the hash is as defined in the "Group Hash to Pallas and Vesta" section of the protocol spec, and that it is intended to follow the Internet Draft (but the protocol spec takes precedence in the case of any discrepancy).

Reply:
IOW, the formatting isn't what determines what is normative. Only the plain text of the specification determines what is normative (preferably by using RFC 2119 keywords).